README.md

```diff
@@ -1822,6 +1822,7 @@ The following configuration is available in `disco_elasticache.ini` to configure
     port=6379
     parameter_group=default.redis2.8
     num_nodes=2
+    maintenance_window=sun:5:00-sun:09:00
```

Comment:
To echo @pchand-amplify 's earlier comment, why are we setting an example window of 4 hours instead of 1 hour? Is a wider window encouraged?